### PR TITLE
Add tests for html2wt as described in T75955

### PR DESCRIPTION
**Pull request by earldouglas**
**NOTE: Don't merge this yet -- it depends on a yet-to-be-released version of Parsoid.  It totally passes locally though, I swear.**

This includes tests for:
 * html -> wt
 * html -> pagebundle
 * html -> html

This currently depends on running a patched version of Parsoid locally.

### CODE REVIEW

File: lib/filters/global/parsoid.js [hunk 1]
```diff
@@ -109,6 +130,15 @@ module.exports = function (conf) {
             },
             '/v1/{domain}/pages/{key}/data-parsoid/{revision}': {
                 get: { request_handler: getDataParsoidRevision }
+            },
+            '/v1/{domain}/transform/{from}/to/{to}': {
+                post: { request_handler: parsoidConvert(conf.parsoidHost) }
+            },
+            '/v1/{domain}/transform/{from}/to/{to}/{title}': {
```
Comment (gwicke) [inline lib/filters/global/parsoid.js]: We should eventually narrow this down to html/to/wikitext and perhaps wikitext/to/html.

Comment (earldouglas) [inline lib/filters/global/parsoid.js]: :+1: Shall I do that on this PR?

Comment (gwicke) [inline lib/filters/global/parsoid.js]: I think it's fine to do that along with the swagger spec stuff.

File: test/features/parsoid/html2wt.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+var assert = require('../../utils/assert.js');
+var preq = require('preq');
+var fs = require('fs');
+
+// read the contents of a spec file as JSON or raw HTML
+function readFile(filename) {
+    var contents = fs.readFileSync(filename, { encoding: 'utf8' });
+
+    // readFileSync seems to append a line feed
```
Comment (gwicke) [inline test/features/parsoid/html2wt.js]: I think most editors on unix store files with a trailing newline.

**Pull request merged**